web/apps/labelstudio/src/pages/WebhookPage/WebhookDetail.jsx

```diff
@@ -253,7 +253,7 @@ const WebhookDetail = ({ webhook, webhooksInfo, fetchWebhooks, onBack, onSelectA
                     })
                   }
                 >
-                  Delete Webhook...
+                  Delete Webhook
```

Comment:
Just want to highlight that we used this UX pattern here, when ellisis represents action not being done immediately, but I'm not sure how popular is this pattern nowdays

Reply:
I'd counter argue this conflicts with css text truncation, and probably something that is not well-known by the broader community. We can always revisit this, with the eyes of @caitlinwheeless!